packages/external-db-authorization/lib/strategies/gcp_strategy.js

```diff
@@ -12,8 +12,27 @@ class GcpStrategy {
     return new GoogleStrategy(this.options, this.verify)
   }
 
-  verify(AccessToken, tokenSecret, profile, done) {
-        done(null, profile)
+  verify(accessToken, tokenSecret, profile, cb) {
+    const usersEmail = profile._json.email
+
+    const getProjectOwners = async() => {
```

Comment:
Couldn't move this function to `gcpStrategy` class,  verify function is being used by passport package, and If i will move this function to the class I will have to use `this` keyword that will be **different** when passport will run this function ...